www/include/configuration/configObject/host/resolveHostName.php

```diff
@@ -57,4 +57,9 @@
 /**
  * Resolving host name
  */
-echo gethostbyname($_GET['hostName']);
+$hostName = str_replace(
+    array('<', '>'),
+    array('&lt;', '&gt;'),
+    $_GET['hostName']
+);
+echo gethostbyname($hostName);
```

Comment:
should not be better echo htmlentities(gethostbyname($hostName)); ?

Reply:
If we do that this will be the result:
![screenshot from 2018-12-12 10-56-55](https://user-images.githubusercontent.com/365020/49858084-b8dedf80-fdfc-11e8-9233-dcc85730b127.png)

I propose to strip the tags and remove all chars that are not compatible with the hostname.